metadata.yaml

```diff
@@ -55,6 +55,12 @@ requires:
     description: |
       Ingress-per-app, to load-balance across multiple units of the coordinator.
 
+  tracing:
+    interface: tracing
+    limit: 1
+    description: |
+      Enables sending traces for charm troubleshooting.
```

Comment:
for now this is fine, but keep in mind some day we might use tracing to monitor more than the charm itself (nginx?)

e.g. send the tracing info forward to the workers so we can send mimir traces to tempo, and the mirmir_worker charm as well.

Reply:
True, and that's absolutely wanted for future. Changed the description to be more universal.